### PR TITLE
✨ Add Name input component

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Name input component
- Add "type" property to Confirm button component

Deployment: https://sisu-pipelin-input-g7uhlyxmoe9.herokuapp.com/storybook/index.html?path=/story/common-input--name
Part of issue #1

### CODE REVIEW

File: components/input/NameInput.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const [nameStatus, setNameStatus] = useState("");
+
+  const checkValueLength = (event) => {
+    const valueLength = event.target.value.length;
+    if (valueLength >= 11) {
+      setNameStatus("Character limit reached!");
+    } else {
+      setNameStatus("");
+    }
+  };
```
Comment (phlgr) [inline components/input/NameInput.tsx]:
```suggestion
  const [isMaxLength, setIsMaxLength] = useState(false);
  
  const checkValueLength = (event) => {
    setIsMaxLength(event.target.value.length > 11);
  };
```

File: components/input/NameInput.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <>
+      <div className={styles.container}>
+        <input
+          className={styles.input}
+          type="text"
+          placeholder="Enter Name"
+          maxLength={11}
+          onChange={(event) => checkValueLength(event)}
+        />
+        <p className={styles.status}>{nameStatus}</p>
```
Comment (phlgr) [inline components/input/NameInput.tsx]:
```suggestion
        { isMaxLength && <p className={styles.status}>Character limit reached!</p>}
```

**Review by phlgr — COMMENTED** [review]
Just a minor improvement 🚀

File: components/input/NameInput.tsx (hunk 1 above)
Comment (fre-ben) [reply components/input/NameInput.tsx]: I'll try this, looks good in theory :)

**Pull request merged**